src/Sarif.Viewer.VisualStudio/Models/SarifErrorListItem.cs

```diff
@@ -673,29 +673,28 @@ internal IEnumerable<string> GetCodeSnippets()
             return this.Locations.Select(location => location.ExtractSnippet(regionsCache));
         }
 
-        internal (string first, string second) SplitMessageText(string fullText, int maxLength = 120)
+        internal (string first, string second) SplitMessageText(string fullText, int maxLength = 165)
         {
-            string shortText = ExtensionMethods.GetFirstSentence(fullText);
+            // remove line breakers
+            fullText = fullText.Replace(Environment.NewLine, " ").Replace("\r", " ").Replace("\n", " ");
```

Comment:
>fullText = fullText.Replace(Environment.NewLine, " ").Replace("\r", " ").Replace("\n", " "); [](start = 12, length = 92)

I'm not sure about this.

Reply:
Environment.NewLine (\r\n) is windows sty, \n and \r are unix/mac style line breakers. I see sarif-sdk code handles all of 3 so I did the similar here.

---
In reply to: [573064240](https://github.com/microsoft/sarif-visualstudio-extension/pull/371#discussion_r573064240) [](ancestors = 573064240)